### ISSUE TITLE
Close exe if PID passed in

### ISSUE THREAD

**Issue by jchannon** [OmniSharp/omnisharp-server#187]
Is it possible to add the ability to kill itself once the pid of sublime is killed somehow?

**Comment by jchannon** [comment]
I assume its this line? https://github.com/OmniSharp/omnisharp-roslyn/blob/master/src/OmniSharp/Program.cs#L119

**Comment by nosami** [comment]
Yeah... and this https://github.com/OmniSharp/omnisharp-roslyn/blob/db74baa796e2dd65c1fd2751c601a02f91617a87/src/OmniSharp/Utilities/ProcessExtensions.cs

**Comment by jchannon** [comment]
See #188
